packages/video_player/video_player_avfoundation/darwin/RunnerTests/VideoPlayerTests.m

```diff
@@ -664,6 +664,30 @@ - (void)testSeekToleranceWhenSeekingToEnd {
   XCTAssertNil(error);
   XCTAssertEqual(avPlayer.volume, 0.1f);
 
+  // Set picture-in-picture
```

Comment:
My [earlier comment about this testing](https://github.com/flutter/packages/pull/3500#discussion_r1307602783) was never resolved.